app/Filament/Resources/IdentityResource/Pages/ListIdentities.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I got to the Identity related Filament pages by first going to the interpretations one and selecting "identities" from the sidebar. Is there any other way to get there?

Reply:
@jobara I didn't think I should add it yet until we discussed the issues related to your subsequent comment.